exporters/stdout/stdouttrace/example_test.go

```diff
@@ -91,5 +91,8 @@ func Example() {
 		}
 	}()
 
-	log.Println("the answer is", add(ctx, multiply(ctx, multiply(ctx, 2, 2), 10), 2))
+	ctx, ans := multiply(ctx, 2, 2)
+	ctx, ans = multiply(ctx, ans, 10)
+	ctx, ans = add(ctx, ans, 2)
```

Comment:
I find returning a `ctx` pretty unusual (code smell). Why not creating a parent span like:



```suggestion
	ctx, span = tracer.Start(ctx, "Calculation")
	defer span.End()
	ans := multiply(ctx, 2, 2)
	ans = multiply(ctx, ans, 10)
	ans = add(ctx, ans, 2)
```